docs/source/getting_started/active_learning.rst

```diff
@@ -180,13 +180,26 @@ Image Classification
 ^^^^^^^^^^^^^^^^^^^^^
 Use this scorer when working on a classification problem (binary or multiclass).
 
-Currently, the following scorers are available:
 
-- **prediction-margin** uses the margin between 1.0 and the highest confidence 
-  prediction. Use this scorer to select images where the model is insecure.
+Currently we offer three uncertainty scorers,which are taken from
+http://burrsettles.com/pub/settles.activelearning.pdf, Section 3.1, page 12f
+and also explained in https://towardsdatascience.com/uncertainty-sampling-cheatsheet-ec57bc067c0b
+They all have in common, that the score is highest if all classes have the
+same confidence and are 0 if the model assigns 100% probability to a single class.
+The differ in the number of class confidences they take into account.
+
+- **uncertainty_least_confidence**:
```

Comment:
Why not just call it `least_confidence`?
I think that is precise enough and simpler to remember.